### PR TITLE
Pool latency histograms for sessions to avoid GC hit

### PR DESCRIPTION
**Pull request by badrishc**

### CODE REVIEW

**Comment by yrajas** [comment]
Instance of LongHistogram in LatencyMetricsEntry needs a Return as well to return the shared ArrayPool?

File: libs/server/Metrics/Latency/GarnetLatencyMetricsSession.cs [hunk 1]
```diff
@@ -24,6 +24,16 @@ public GarnetLatencyMetricsSession(GarnetServerMonitor monitor)
             Init();
         }
 
+        public void Return()
+        {
+            foreach (var cmd in defaultLatencyTypes)
+            {
```
Comment (yrajas) [inline libs/server/Metrics/Latency/GarnetLatencyMetricsSession.cs:30]: Better to have a Return method within LatencyMetricsEntrySession that this method can invoke to maintain the abstraction? That way, invoking Return will become obvious to any future users of LatencyMetricsEntrySession.

Comment (badrishc) [reply libs/server/Metrics/Latency/GarnetLatencyMetricsSession.cs:30]: Done

**Comment by badrishc** [comment]
> Instance of LongHistogram in LatencyMetricsEntry needs a Return as well to return the shared ArrayPool?

Added.

**Review by yrajas — APPROVED** [review]

**Pull request merged**